rwlib/source/loaders/LoaderDFF.cpp

```diff
@@ -359,7 +359,7 @@ void LoaderDFF::readTexture(Geometry::Material &material,
 
     TextureData::Handle textureinst =
         texturelookup ? texturelookup(name, alpha) : nullptr;
-    material.textures.push_back({name, alpha, textureinst});
+    material.textures.emplace_back(name, alpha, textureinst);
```

Comment:
You should be using `std::move` here too, as now you're passing your strings by value. Looked up the full function and looks like moving those strings away is perfectly safe.